examples/tensorflow/sagemaker_official_container/horovod_estimator_mnist.py

```diff
@@ -35,7 +35,7 @@
 import tensorflow as tf
 from tensorflow import keras
 
-tf.logging.set_verbosity(tf.logging.INFO)
+tf.compat.v1.logging.set_verbosity(tf.compat.v1.logging.INFO)
```

Comment:
why this change?

Reply:
`tf.logging` has been moved to `tf.compat.v1.logging` as of TF 1.14 [link](https://stackoverflow.com/questions/38073432/how-to-suppress-verbose-tensorflow-logging) .

But this should ideally be done outside of this PR.
I can revert this.